frameworks/CFML/cfml-lucee/cfml-lucee.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ortussolutions/commandbox:latest
```

Comment:
Can you pin this down to a specific version, like 3.0.1

Reply:
Pinned to 3.2.1